### PR TITLE
[NO_TICKET] Update html examples

### PR DESCRIPTION
**Pull request by nichia**
### Summary
Update the html examples (article, form and responsive) to reference css file with the cmsds v2 folder structure.

### CODE REVIEW

File: packages/design-system/src/styles/settings/variables/_build.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Paths
-$font-path: './fonts' !default;
-$image-path: './images' !default;
+$font-path: '../fonts' !default;
+$image-path: '../images' !default;
```
Comment (line47) [inline packages/design-system/src/styles/settings/variables/_build.scss:3]: Can you talk about this change here? I reverted back to the single `./` and didn't see a difference. What is this change fixing?

Comment (nichia) [reply packages/design-system/src/styles/settings/variables/_build.scss:3]: The `../` fixes those error msgs on Website when viewing the dev tool's console 
Screen Shot 2020-07-27 at 10 15 18 AM

Comment (nichia) [reply packages/design-system/src/styles/settings/variables/_build.scss:3]: as the `src/font` & `src/image` folders are on the same level as `src/styles`

Comment (nichia) [reply packages/design-system/src/styles/settings/variables/_build.scss:3]: I'll check again, looks like maybe it did not fix that.

Comment (nichia) [reply packages/design-system/src/styles/settings/variables/_build.scss:3]: Checked - yes, it fixed the error msgs (I forgot to yarn build again after changing the variables).

**Comment by nichia** [comment]
Transferred to pr WNMGDS-495 Update CMSDS examples.

**Pull request closed without merging**